contracts/staking/interfaces/IValidatorManager.sol

```diff
@@ -105,9 +105,6 @@ interface IValidatorManager {
      * {registrationExpiry} being reached.
      * @param messageIndex The index of the Warp message to be received providing the proof the validation is not active
      * and never will be active on the P-Chain.
-     * @param setWeightMessageType Whether or not the message type is a SetValidatorWeight message, or a
-     * SubnetValidatorRegistration message (with valid set to false). Both message types are valid for ending
```

Comment:
I think this doc block should still mention the expectation that the message is a `SubnetValidatorRegistration` message, I guess in the `@param` description

Reply:
I think the Warp message format is an implementation detail that doesn't need to be mentioned in the interface. For consistency, it's also not mentioned in the `completeValidatorRegistration` description.